default.build

```diff
@@ -170,9 +170,12 @@
   <target name="build_windows10" description="Build Windows 10 UAP" depends="clean">
     <if test="${os == 'Win32NT'}">
       <if test="${file::exists('c:\Program Files (x86)\MSBuild\Microsoft\WindowsXaml\v14.0\Microsoft.Windows.UI.Xaml.CSharp.targets')}">
-        <exec program="Protobuild" commandline="-generate WindowsUAP" />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU"' />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU"' />
+        <exec program="Protobuild" commandline="-generate WindowsUniversal" />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
```

Comment:
> Another issue I just noticed on the build server. Although the ThirdParty/Kickstart and ThirdParty/Dependencies exist on the build agent... Protobuild is git checking out another copy. This seems like it is because Protobuild looks for the existence of the .git in those submodules... which does not exist because that isn't how TeamCity works.

How do these references work?  If TeamCity isn't checking out the submodule, then nothing should be building...